scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -198,7 +240,7 @@ def _get_caseroot2(self):
         casename1 = self._case1.get_value("CASE")
         caseroot1 = self._case1.get_value("CASEROOT")
 
-        casename2 = "{}.{}".format(casename1, self._run_two_suffix)
+        casename2 = "{}.{}".format(casename1, self._run_two_suffix) if self._separate_runs else casename1
```

Comment:
Why does everyone hate ternaries? :)

I'll leave this to the broader group since I've used this pattern in dozens of places. I personally like conciseness of a ternary for simple if/else assignments like this. 4 lines of code seems excessive to express such a simple concept.